Modules.md

```diff
@@ -11,6 +16,13 @@ is called a **module**. A module contains:
 Sections declare their type and byte-length. Sections with unknown types are
 silently ignored.
 
+An instance contains:
+* a [linear memory](AstSemantics.md#linear-memory);
+* [global variable](AstSemantics.md#global-variables) state;
+* (when [threading](PostMVP.md#threads) is added) thread-local state;
+* (when [dynamic linking](DynamicLinking.md) is added) the code of multiple modules
+  that have been linked into the same instance;
```

Comment:
What do you mean by "code"? The instance doesn't contain the main module's code, so mentioning the code from loaded modules is asymmetric.

Maybe: import/export tables, and globals?